README.md

```diff
@@ -124,6 +124,61 @@ cd cmake-build-release
 ./componentize.sh ../tests/smoke.js
 ```
 
+6. Run project-specific commands using `just`
+
+The justfile provides a streamlined interface for executing common project-specific tasks. To install just, you can use the command:
+
+``` shell
+cargo install just
+```
+
+Alternatively, refer to the official [installation instructions](https://github.com/casey/just?tab=readme-ov-file#installation) for your specific system.
+
+Once installed, navigate to the project directory and run `just` commands as needed. For instance, the following commands will configure a default `cmake-build-debug` directory and build the project.
+
+``` shell
+just build
+```
+
+To build and run integration tests run:
+
+``` shell
+just test
+```
+
+To build and run Web Platform Tests run:
+
+``` shell
+just wpt-test                                   # run all tests
+just wpt-test console/console-log-symbol.any.js # run specific test
+```
+
+The default build directory (`cmake-build-debug`) and build flavor (`debug`) can be overridden via command-line arguments. For example, the following command configures CMake to use `mybuilddir` as the build directory and sets the build type to `release`:
+
+``` shell
+just builddir=mybuilddir flavour=release build
```

Comment:
We should probably use american english - `flavor`. Alternatively use a term like `mode`.

Reply:
It's probably also worth noting here that the builddir is automatically changed depending on the build mode, so introduce the build mode customization first, and then separately introduce customizing the build dir.

Reply:
OK, I've changed the flavour to mode.